exercises/random_number.cpp

```diff
@@ -4,3 +4,24 @@
   Output:
   The random number is: 4
 */
+
+
+
+#include<iostream>
+#include<cstdlib>
+#include<climits>
+#include<ctime>
+
+using namespace std;
+
+
+int main() {
+  
+  srand(time(0));
+
+  //Generating the random number
```

Comment:
metti uno spazio dopo `//`
```suggestion
  // Generating the random number
```